<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vpComputeColorRangeDialog</class>
 <widget class="QDialog" name="vpComputeColorRangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute Color Range</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="computeCurrentFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>211</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Frame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="computeAllFrames">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>187</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>All Frames</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Computed Values</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="widthValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Center:</string>
    </property>
   </widget>
   <widget class="QLabel" name="centerValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>116</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vpComputeColorRangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vpComputeColorRangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
